mbrl/models/gaussian_mlp.py

```diff
@@ -89,19 +93,27 @@ def __init__(
         self.in_size = in_size
         self.out_size = out_size
 
-        activation_cls = nn.SiLU if use_silu else nn.ReLU
+        if use_silu:
+            warnings.warn("use_silu is deprecated and will be removed.
+                          Use activation_name = 'SiLU' instead.")
+            activation_func = nn.SiLU()
+
+        if activation_name == "ReLU" and not use_silu:
```

Comment:
I think it's fine if the new mechanism is always used when if `use_silu = False`, so we can skip this special case for `nn.ReLU()` (as long as the default when nothing is specified is a relu activation).